http/cookie.ts

```diff
@@ -296,7 +299,7 @@ export function setCookie(headers: Headers, cookie: Cookie) {
 export function deleteCookie(
   headers: Headers,
   name: string,
-  attributes?: { path?: string; domain?: string },
+  attributes?: Omit<Cookie, "name" | "value">,
```

Comment:
I don't think this is correct. As far as I tested, `sameSite` attribute didn't have effect in `deleteCookie` (You can delete sameSite=Lax cookie without specifying `sameSite=Lax` when deleting it)

Reply:
Fixed. PTAL.

Reply:
LGTM

Reply:
> I don't think this is correct. As far as I tested, `sameSite` attribute didn't have effect in `deleteCookie` (You can delete sameSite=Lax cookie without specifying `sameSite=Lax` when deleting it)

It work on localhost, but don't when deployed. Using Deno Deploy